inflammation/models.py

```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
+    devs = []
+    for entry in data:
+        devs.append((entry - mmm) * (entry - mmm))
+
+    s_dev2 = sum(devs) / len(data)
+    return {'standard deviation': s_dev2}
```

Comment:
Solution says: Have the function return the data, rather than having the graph name (a view layer consideration) leak into the model code. 

I don't really get this.

Reply:
After talking about it, I get it now. The function should return a numpy array and not a dictionary. This needs a change in the inflammation-analysis.py, ln 24, and include the graph name for the standard deviation similarly to the rest.

Reply:
Fixed in 618d472